### PR TITLE
added address and song endpoint data to homepage

### LINKED ISSUE #107: Adding endpoints information in the homepage.
**Describe the feature**
Add following endpoints information on the homepage

### /address endpoints

- [ ] `/address?city=abc`
- [ ] `/address?state=xyz`
- [ ] `/address?country=def`
- [ ] `/address?city=abc&state=xyz&country=def`

### /songs endpoints

- [ ] `/songs?artist=abc`
- [ ] `/songs?genre=def`
- [ ] `/songs?language=ghi`
- [ ] `/songs?artist=abc&genre=def&language=ghi`

**Are you working on this feature? (Yes)**

### PR DESCRIPTION
**Pull request by ritik307**
# Related Issue

-   closes #107 

# Proposed Changes
### /address endpoints

- [x] `/address?city=abc`
- [x] `/address?state=xyz`
- [x] `/address?country=def`
- [x] `/address?city=abc&state=xyz&country=def`

### /songs endpoints

- [x] `/songs?artist=abc`
- [x] `/songs?genre=def`
- [x] `/songs?language=ghi`
- [x] `/songs?artist=abc&genre=def&language=ghi`

# Checklist

-   [x] Tests
-   [x] Translations
-   [x] Run Eslint

### CODE REVIEW

**Comment by ritik307** [comment]
@ShauryaAg @paramsiddharth Plz review!

File: src/views/data.ts [hunk 1]
```diff
@@ -177,7 +177,8 @@ const data = [
     },
     {
         heading: "FoodPhotos",
-        desc: "The foodphotos route returns photos of food item with their source link,name and taste.",
+        desc:
+            "The foodphotos route returns photos of food item with their source link,name and taste.",
```
Comment (ShauryaAg) [inline src/views/data.ts:181]: Why was this change made?

File: src/views/data.ts [hunk 2]
```diff
@@ -219,7 +220,8 @@ const data = [
     },
     {
         heading: "Animal",
-        desc: "The animal route returns dummy data realted to animals with their type,breed and photos.",
+        desc:
+            "The animal route returns dummy data realted to animals with their type,breed and photos.",
```
Comment (ShauryaAg) [inline src/views/data.ts:224]: Why was this change made?

File: src/views/data.ts [hunk 3]
```diff
@@ -449,6 +537,98 @@ const data = [
...
+                      "link": "https://www.youtube.com/watch?v=VNs_cCtdbPc"
+                  }
+              
+              ]`,
+            },
+            {
+                route: "GET /songs?artist=A.R. Rahman&genre=sufi&language=urdu",
+                content: "Return dummy songs data after chaining multiple song query parameter.",
+                output: `[
+                {
```
Comment (ShauryaAg) [inline src/views/data.ts:622]: Add a route where you are querying only based on two things. Say `genre` and `language`.

File: src/views/data.ts [hunk 4]
```diff
@@ -389,6 +391,93 @@ const data = [
...
+                      "pincode": "400067"
+                  }
+              
+              ]`,
+            },
+            {
+                route: "GET /address?city=Mumbai&state=Maharastra&country=India",
+                content: "Return dummy data after chaining multiple address query parameter.",
+                output: `[
+
```
Comment (ShauryaAg) [inline src/views/data.ts:456]: Add a route that queries only two fields. Say `city` and `state`

**Review by ShauryaAg — CHANGES_REQUESTED** [review]

File: src/views/data.ts (hunk 4 above)
Comment (ritik307) [reply src/views/data.ts]: Done! Plz review

File: src/views/data.ts (hunk 3 above)
Comment (ritik307) [reply src/views/data.ts]: Done! Plz review

**Comment by ritik307** [comment]
@ShauryaAg Plz review

**Review by ShauryaAg — APPROVED** [review]

**Pull request merged**